src/uipath/eval/coded_evaluators/contains_evaluator.py

```diff
@@ -0,0 +1,55 @@
+"""Contains evaluator for agent outputs."""
+
+from ..models import AgentExecution, EvaluationResult, NumericEvaluationResult
+from .output_evaluator import (
+    OutputEvaluationCriteria,
+    OutputEvaluator,
+    OutputEvaluatorConfig,
+)
+
+
+class ContainsEvaluatorConfig(OutputEvaluatorConfig):
+    """Configuration for the exact match evaluator."""
+
+    name: str = "ContainsEvaluator"
+    case_sensitive: bool = False
+    negated: bool = False
+
+
+class ContainsEvaluator(OutputEvaluator[ContainsEvaluatorConfig]):
+    """Evaluator that checks if the actual output contains the expected output.
+
+    This evaluator returns True if the actual output contains the expected output,
+    and False otherwise. It supports case sensitivity and negation options.
+    """
+
+    async def evaluate(
+        self,
+        agent_execution: AgentExecution,
+        evaluation_criteria: OutputEvaluationCriteria,
```

Comment:
Evaluation Criteria would need to be defined for each Evaluator

Contains will have searchText we don't have a type representing that right now.